index.js

```diff
@@ -16,6 +17,18 @@ function lint(textEditor) {
 
 	const dir = pkgDir.sync(path.dirname(filePath));
 
+	// no package.json
+	if (!dir) {
+		return [];
+	}
+
+	// check if package.json has 'xo'
+	const rgx = new RegExp('xo', 'g');
+	const pkg = fs.readFileSync(path.join(dir, 'package.json'), 'utf8');
```

Comment:
Just use https://github.com/sindresorhus/load-json-file and check for a `xo` dependency key in the object. Using a regex is way too fragile. It would match even if a package.json would have a word like `claxon` in the description field.